src/utils/composeStores.js

```diff
@@ -1,8 +1,17 @@
 import mapValues from '../utils/mapValues';
 import pick from '../utils/pick';
+import invariant from 'invariant';
 
 export default function composeStores(stores) {
   const finalStores = pick(stores, (val) => typeof val === 'function');
+
+  Object.keys(finalStores).forEach(key => {
+    invariant(
+        typeof finalStores[key]({}, {}) !== 'undefined',
+        'The return value of `%s` reducer function must not be undefined.',
```

Comment:
Until we adopt the new naming everywhere let's change the message to

"Your `%s` Store returns undefined when given an unknown action. Change it to return the `state` given to it for any unknown actions."